Screenrecorder.py

```diff
@@ -0,0 +1,47 @@
+import cv2
+import numpy as np
```

Comment:
Please follow the folder structure as mentioned in the `README.md`.

You should move this file under `Task 1` folder, and in its own folder within it.

Something like

`Task 1/ScreenRecorder/ScreenRecorder.py`

Please also add a `Question.md` file.